src/utils/logger.utils.ts

```diff
@@ -0,0 +1,48 @@
+import winston from 'winston';
+
+//Winston aliases for readability
+const combine = winston.format.combine;
+const timestamp = winston.format.timestamp;
+const prettyPrint = winston.format.prettyPrint;
+const colorize =  winston.format.colorize;
+const align = winston.format.align;
+const printf = winston.format.printf;
+
+const alignedWithColorsAndTime = winston.format.combine(
+  colorize(),
+  timestamp(),
+  align(),
+  printf((info) => {
+    const {
+      timestamp, level, message, ...args
+    } = info;
+
+    const ts = timestamp.slice(0, 19).replace('T', ' ');
```

Comment:
explain what you're doing here

Reply:
it timestamps the log in printable format. this is the out of the box timestamp implementation from winston examples